README.md

```diff
@@ -144,8 +144,10 @@ Runs:
 
 * __scssLint(gulp, config)__ Config accepts:
 	- scssLintPath: `String` Path to your custom 'scss-lint.yml' config file. (Default: 'origami-build-tools/config/scss-lint.yml') _This may be set for product development, but developers of Origami-compliant components are required to accept the default_
+	- excludeFiles `Array` e.g. `['!**/demo.scss']`
 * __jsHint(gulp, config)__ Config accepts:
 	- jsHintPath: `String` Path to your custom jsHint config file. (Default: 'origami-build-tools/config/jshint.json' _This may be set for product development, but developers of Origami-compliant components are required to accept the default_
+	- excludeFiles `Array` e.g. `['!**/demo.js']`
```

Comment:
Right. I guess it's simple enough (a concat from a global array…) for not requiring tests.